cpp/modmesh/view/RCameraController.cpp

```diff
@@ -212,6 +212,18 @@ void RCameraInputListener::initKeyboardListeners() const
     m_ty_axis->addInput(m_keyboard_ty_neg_input);
 }
 
+void CameraController::reset()
+{
+    camera()->lens()->setPerspectiveProjection(45.0f, camera()->lens()->aspectRatio(), 0.1f, 1000.0f);
```

Comment:
Now don't reset `aspectRatio`. Because if resize a window and reset, then mesh will look strange. Qt automatically already updates `aspectRation` when window is resized

Reply:
Better to have a comment to explain what is 45.0, 0.1, and 1000.0. Or, you can have `constexpr angle = 45.0f`, which can better explain the code.

Reply:
It is a good idea to add local `constexpr` for constants.  It will be better document than comments.

Angles need special treatment.  If it's in degree, the (local) name should contain it like `front_deg`.  If it's in radian, the name should contain it accordingly.

It's impossible to exclusively use either degree or radian for angles.  Both are needed by different libraries.  Conversion is a must.  Local variable names should make it clear which unit is used.

---

Names for member data and function parameters of angles are trickier.